projects/addon-commerce/components/money/pipes/fraction-part.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import {Inject, Pipe, PipeTransform} from '@angular/core';
+import {TUI_NUMBER_FORMAT, TuiDecimalT, TuiNumberFormatSettings} from '@taiga-ui/core';
+
+@Pipe({name: 'tuiFractionPart'})
+export class TuiFractionPartPipe implements PipeTransform {
+    constructor(
+        @Inject(TUI_NUMBER_FORMAT) private readonly numberFormat: TuiNumberFormatSettings,
+    ) {}
+
+    transform(value: number, decimal: TuiDecimalT, precision: number): string {
+        const fraction = value.toFixed(precision).split('.')[1];
+
+        return decimal === 'never' ||
+            (parseInt(fraction, 10) === 0 && decimal !== 'always')
```

Comment:
Let's move it to `const shouldShow = decimal === 'always' || !!parseInt(fraction, 10)`

Reply:
fixed